<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Clear">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MakePolygon">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Make Polygon</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ConvexHull">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Convex Hull</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Triangulate">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Triangulate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SurfaceArea">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Surface Area</string>
    </property>
   </widget>
   <widget class="QLabel" name="SurfaceAreaLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>441</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: red;</string>
    </property>
    <property name="text">
     <string>Surface area of selected polygon: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="SmallestCircle">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Smallest Circle</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRestart">
    <property name="title">
     <string>Menu</string>
    </property>
   </widget>
   <addaction name="menuRestart"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRestart">
   <property name="text">
    <string>Restart</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
